### PR TITLE
[PBP-86] Add licensing headers to files in metatemplates

### PR DESCRIPTION
**Pull request by gromakovsky**
Problem: according to our current best practices we want each text
file to contain a header which specifies its licensing information.
However, files in this repo do not have these headers.

Solution: add headers to all files that are supposed to be copied
to a new repo when it is created. Exception: files in `reuse` which
apparently do not have to have these headers.

### CODE REVIEW

File: templates/.github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<!--
+ - SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: LicenseRef-MPL-2.0
```
Comment (kirelagin) [inline templates/.github/CONTRIBUTING.md]: There is not need for `LicenseRef-` here as we use a standard license.

File: templates/.github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+<!--
+ - SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
```
Comment (kirelagin) [inline templates/.github/CONTRIBUTING.md]: I think a somewhat standard thing is to align the dash with the last dash of the first line 🤔.

File: templates/haskell/.editorconfig [hunk 3]
```diff
@@ -1,3 +1,9 @@
+<!--
```
Comment (kirelagin) [inline templates/haskell/.editorconfig]: This comment syntax doesn’t look right :).

**Comment by gromakovsky** [comment]
Closing due to #11.

**Pull request closed without merging**